lib/cc/engine/analyzers/violation.rb

```diff
@@ -91,19 +112,15 @@ def fingerprint
           digest << "-"
           digest << current_sexp.mass.to_s
           digest << "-"
-          digest << occurrences.to_s
+          digest << occurrences.length.to_s
```

Comment:
I would've guessed this logic is in the compare view(s), but I can't find it. The only `uniq`-ing by fingerprint seems to be in `SmellsCounter`:

https://github.com/codeclimate/app/blob/master/app/models/smells_counter.rb#L50

So perhaps that's the only place we want special behavior: in the category counts.